test/e2e/case38_install_operator_test.go

```diff
@@ -17,7 +17,7 @@ var _ = Describe("Test installing an operator from OperatorPolicy", Ordered, fun
 		opPolTestNS      = "operator-policy-testns"
 		parentPolicyYAML = "../resources/case38_operator_install/parent-policy.yaml"
 		parentPolicyName = "parent-policy"
-		opPolTimeout     = 10
+		opPolTimeout     = 60
```

Comment:
👀 what's taking up to 60 seconds? I'd rather we adjust it for that specific test somehow. Maybe a separate `Eventually` for whatever that test is waiting a while for, before the main `check`.

It looks like this is only (currently) used in `Eventually` kinds of checks, so keeping this change wouldn't be the end of the world.